geoips/plugins/yaml/workflows/geostationary_dmw/windbarbs_dmw_low.yaml

```diff
@@ -0,0 +1,45 @@
+interface: workflows
+family: order_based
+name: ob_windbarbs_dmw_low
+docstring: |
+  The windbarbs workflow configuration.
+spec:
+  steps:
+    - algorithm:
+        name: windbarbs_dmw
+        arguments:
+          pressure_level_range: [701, 1013.25]
+          output_data_range: [0, 200]
+          input_units: m s-1
+          output_units: kts
+          min_outbounds: crop
+          max_outbounds: crop
+          norm: null
+          inverse: null
+    - coverage_checker:
+        name: windbarbs
+        arguments: {}
+    - output_formatter:
+        name: imagery_windbarbs
+        arguments:
+          colormapper:
+            name: matplotlib_linear_norm
+            arguments:
+              data_range: [0, 200]
+              cbar_ticks: [0, 20, 40, 60, 80, 100, 120, 140, 160, 180, 200]
+              cmap_name: viridis
+              cbar_label: "Derived Motion Wind (kts)\nLow Level: 1013.25 - 701 hPa"
+          windbarb_plotter:
+            name: imagery_windbarbs
+            arguments:
+              thinning: 1  # skip data points
+              length: 5.0
+              width: 1.5
+              sizes_dict:
+                height: 0.7
+                spacing: 0.3
+              rain_size: 0
+          filename_formatter:
+            name: geoips_fname
+            arguments:
+              suffix: ".png"
```

Comment:
Oh, right, we already have the output_formatter copied to several workflows.

Maybe in the end we will want to organize the workflow YAMLs similarly to how we have products / product defaults / etc organized now?  Just to organize and categorize the different "types" of workflows ?  It shouldn't matter how they are organized, but it might help keep things straight and encourage modular development and re-use?

Reply:
Yeah, that would probably be good to do in the future. I think it can wait until we see how the OBP evolves, though. Right now, the output formatters aren't actually being called. The only plugins that actually get called right now are the Reader plugins. Embedded workflow plugins are loaded but not run. Everything else just reports that it is being skipped.

Reply:
As long as the intent is to make it possible to lay things out exactly how they are laid out now, it is fine.  Just want to make sure we don't lock ourselves into a corner.

Reply:
Yeah, the idea is to use translators to get from the current format to the new format. We'll be implementing those later, but I don't anticipate any major problems with that.